Source/WebCore/rendering/RenderGrid.cpp

```diff
@@ -2241,46 +2241,37 @@ ContentPosition static resolveContentDistributionFallback(ContentDistribution di
     return ContentPosition::Normal;
 }
 
-static void contentDistributionOffset(ContentAlignmentData& offset, const LayoutUnit& availableFreeSpace, ContentPosition& fallbackPosition, ContentDistribution distribution, unsigned numberOfGridTracks)
+static bool computeDistributionOffset(ContentAlignmentData& offset, const LayoutUnit& availableFreeSpace, ContentPosition& fallbackPosition, ContentDistribution distribution, unsigned numberOfGridTracks)
```

Comment:
Please return a `{ No, Yes }` enum class here, it's unclear what the bool means when reading this function

Reply:
How is No/Yes clearer than False/True?

Reply:
You'd be returning and checking against `ContentDistributionIsValid::No` / `ContentDistributionIsValid::Yes` instead of `true` / `false` which is a lot more clear when reading.

The enum class would be defined as: `enum class ContentDistributionIsValid : bool { No, Yes }`

(I'll leave the name of the enum class up to you, the one I gave here was just an example)

Reply:
Fwiw, this is a pretty common pattern in the WebKit codebase: https://searchfox.org/wubkat/search?q=bool+%7B+No%2C+Yes+%7D%3B&path=&case=false&regexp=false

Reply:
OK, I tried that but it got so wordy, I decided to just fold the entire function into its caller, which I had been wanting to do anyway. I think this is easier to follow than splitting out. Lmk if this works for you.